package.json

```diff
@@ -36,7 +36,6 @@
     "format": "prettier-eslint ./bin/*.js ./test/**/*.js ./packages/**/*.js --write",
     "lint:codeOnly": "eslint \"{bin}/**/!(__testfixtures__)/*.js\" \"{bin}/**.js\"",
     "lint": "eslint \"./bin/*.js\" \"./test/**/*.js\" \"packages/**/!(node_modules)/*.ts\"",
-    "postinstall": "node ./bin/opencollective.js",
```

Comment:
After going through Issue, `open collective` banner should only be disabled when we install `webpack-cli` in CI or any env variable id used.